hapi-proto/src/main/java/com/hedera/services/legacy/proto/utils/KeyExpansion.java

```diff
@@ -51,7 +51,7 @@
 public class KeyExpansion {
 
   private static final Logger log = LogManager.getLogger(KeyExpansion.class);
-  public static int KEY_EXPANSION_DEPTH = 100; // recursion level for expansion
+  public static int KEY_EXPANSION_DEPTH = 15; // recursion level for expansion
```

Comment:
Do we have a external property for this in bootstrap.properties file already?

Reply:
Do, we need it ?

Reply:
I don't know. Probably not.